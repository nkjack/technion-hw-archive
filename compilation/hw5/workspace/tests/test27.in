int mod(int lhs, int rhs) {
	rhs = (lhs / rhs) * rhs;
	return lhs - rhs;
}

void main() {
	print("0 mod 11 = ");
	printi(mod(0,11));
	print("\n18 mod 2 = ");
	printi(mod(18,2));
	print("\n20 mod 3 = ");
	printi(mod(20,3));
	print("\n");
}