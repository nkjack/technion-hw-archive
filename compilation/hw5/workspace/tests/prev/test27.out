Loaded: ./exceptions.s
5
8
I hate this course
